preview/index.ts

```diff
@@ -14,68 +18,62 @@ const PREVIEW_DATA_SOURCE_FILE_PATH = './preview/sampleData.csv';
 
 const logFeedback = (
 	versionNumber: string,
-	newsletters: EmailNewsletter[],
+	newsletters: NewsletterResponse[],
 ): void => {
 	console.log({ versionNumber });
-	console.log('INDEX\tVALID\tCurrent\t CANCELLED\tNAME');
+	console.log('INDEX\tVALID\tCANCELLED\tNAME');
```

Comment:
Since we now call logFeedback on the validated NewsletterResponse, they will always be valid and it won't give feedback on any invalid rows. We should probably remove the  "valid" column in the feedback table since it will always be true and might be slightly misleading
```suggestion
	console.log('INDEX\tCANCELLED\tNAME');
```